### PR TITLE
Sass-rails on Sprockets 4.

### PR DESCRIPTION
**Pull request by Andreis13**

### CODE REVIEW

File: lib/sass/rails/cache_store.rb [hunk 1]
```diff
@@ -1,31 +0,0 @@
-require 'sass'
-
-module Sass
-  module Rails
-    class CacheStore < ::Sass::CacheStores::Base
```
Comment (rafaelfranca) [inline lib/sass/rails/cache_store.rb]: Are we using the default cache store right?

Comment (Andreis13) [inline lib/sass/rails/cache_store.rb]: Yes. The one that is defined here: https://github.com/rails/sprockets/blob/master/lib/sprockets/sass_cache_store.rb

Comment (rafaelfranca) [inline lib/sass/rails/cache_store.rb]: Cool!

**Comment by malida** [comment]
Sorry I haven't spoken up earlier, but Andreis13's work affects work toward supporting Sassc instead of the Sass ruby gem. Since there won’t be any template.rb file I think my changes should be applied to sprockets directly, specifically the sass_processor.rb file. Is that correct? If not, where is the new canonical location for this kind of support? I'm unsure whether to put this stuff in sprockets or sass-rails.

**Comment by rafaelfranca** [comment]
@malida I think we should move everything that is not Rails specific to sprockets and keep this project just to integrate things with Rails.

Instead of changing the `SassProcess` directly I think it is better to investigate a new processor that will deal with sassc, but if the code is too similar an option for the processor like we did with the `importer` on this PR would also work.

Does it sounds a good plan?

**Comment by malida** [comment]
It sounds great! Thanks.

**Pull request merged**